BlueprintUICommonControls/Sources/Internal/KeyboardObserver.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  KeyboardObserver.swift
+//  BlueprintUICommonControls
+//
+//  Created by Kyle Van Essen on 2/16/20.
+//
+
+import UIKit
+
+
+protocol KeyboardObserverDelegate : AnyObject
+{
```

Comment:
Why are `{`s on a newline? That's (very) unusual for Swift, and breaks convention with the rest of the project.

Reply:
This is the only right way to write swift and everyone else is wrong!

(It's copied from another repo, will fix... But I still hate it.)